tests/unit/crsAdapter/BewotecExpertAdapter.test.js

```diff
@@ -213,7 +236,7 @@ describe('BewotecExpertAdapter', () => {
             adapter.fetchData().then(() => {
                 done.fail('unexpected result');
             }, (error) => {
-                expect(error.message).toBe('can not establish connection to bewotec data bridge');
+                expect(error.message).toBe('bewotec data bridge window can not be opened');
```

Comment:
not 'open'?

Reply:
no, else it would mean that it is not allowed to be open.